tools/rna_tools/locarna/locarna.xml

```diff
@@ -1,92 +0,0 @@
-<tool id="locarna" name="locarna" version="0.1.0">
-    <description></description>
-    <requirements>
-    <requirement type="package" version="x.y.z">locarna</requirement>
```

Comment:
regarding the mlocarna wrapper of graphclust ... can you please talk to @mmiladi about this.